docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/azure.md

```diff
@@ -100,7 +100,7 @@ This section is valid only for creating clusters with the in-tree cloud provider
    <details id="v2.6.0-cloud-provider-config-file">
      <summary>Example Cloud Provider Config</summary>
 
-    ```yaml
+```yaml
```

Comment:
What's the expected rendered output here? This block doesn't seem to have the same missing indentation as the other blocks. 

The [build is also failing](https://github.com/rancher/rancher-docs/actions/runs/11133756480/job/30940522383?pr=1494) without the existing indentation.